library/std/Cargo.toml

```diff
@@ -6,7 +6,8 @@
 # standard library symbols are preserved
 name = "std"
 version = "0.5.0"
-edition = "2018"
+edition = "2021"
```

Comment:
Bump the edition for the other `library/` crates? I think I left all of them 2018 iirc

Reply:
Ah. Thanks for pointing it out. I thought you had only left the std one.